src/app/content/components/Toolbar/index.tsx

```diff
@@ -86,9 +88,12 @@ class Toolbar extends React.Component<Props, State> {
 
     const showBackToSearchResults = !this.props.searchSidebarOpen && this.props.hasSearchResults;
 
+    const hideFromFocus = this.props.tocOpen === true
+      || (this.props.tocOpen === null && !this.props.searchSidebarOpen);
```

Comment:
I think that's weird that behavior differs depends on toc state because when toc is open user has to tab through all buttons to get to the Toc and when it is closed it is in another tab order - before search input.
I think we can ask if that should be fixed in this pr or in a separate one

Reply:
https://openstax.slack.com/archives/C8V1XET1R/p1596809983200700